<?xml version="1.0"?>
<launch>

  <param name="traffic_light_1_pose_x" type="double" value="75.0" /> <!--1号红绿灯的x位置，单位：米-->
  <param name="traffic_light_1_pose_y" type="double" value="-58.0" /> <!--1号红绿灯的y位置，单位：米-->
  <param name="traffic_light_2_pose_x" type="double" value="76.0" /> <!--2号红绿灯的x位置，单位：米-->
  <param name="traffic_light_2_pose_y" type="double" value="-57.0" /> <!--2号红绿灯的y位置，单位：米-->
  <param name="traffic_light_radius" type="double" value="30.0" /> <!--红绿灯的半径，小车在这个范围内停车，单位：米-->
  <node name="traffic_light" pkg="traffic_light" type="traffic_light" output="screen"/>

</launch>
